typescript-timers/interval.ts

```diff
@@ -0,0 +1,12 @@
+const h1 = document.querySelector('.countdown-display') as HTMLElement;
+if (!h1) throw new Error('no count');
+
+let currentNumber = 4;
+const intervalId = setInterval(function () {
+  h1.innerText = currentNumber.toString();
+  currentNumber--;
+  if (currentNumber < 0) {
+    clearInterval(intervalId);
+    h1.innerText = '~Earth Beeelooowww Us~';
+  }
+}, 1000);
```

Comment:
This is currently ALWAYS showing the number for the text. I only want to show the number for the text if its greater than 0